.guides/content/Adding-instructional-materials-4f21.md

```diff
@@ -20,7 +20,7 @@ You can assign the project we created earlier or use a Codio supplied project.
 ## Add an Assignment
 5. In your newly created module click **Add Assignment** and then select **Project Based**
 
-![](.guides/img/addassignmentnew.png)
+![.guides/img/addassignmentnew](.guides/img/addassignmentnew.png)
```

Comment:
Instead of the path to the file in the first part just put in something like "Add assignment dropdown"